packages/cli/lib/prompts/selectPublicAppPrompt.js

```diff
@@ -0,0 +1,73 @@
+const { promptUser } = require('./promptUtils');
+const { i18n } = require('../lang');
+const { uiLine } = require('../ui');
+const { logger } = require('@hubspot/local-dev-lib/logger');
+const {
+  fetchPublicAppsForPortal,
+} = require('@hubspot/local-dev-lib/api/appsDev');
+const { EXIT_CODES } = require('../../lib/enums/exitCodes');
+
+const i18nKey = 'lib.prompts.selectPublicAppPrompt';
+
+const fetchPublicAppOptions = async (accountId, accountName) => {
+  try {
+    const publicApps = await fetchPublicAppsForPortal(accountId);
+    const filteredPublicApps = publicApps.filter(
+      app => !app.projectId && !app.sourceId
+    );
+
+    if (!filteredPublicApps.length) {
+      uiLine();
+      logger.error(i18n(`${i18nKey}.errors.noApps`));
+      logger.log(i18n(`${i18nKey}.errors.noAppsMessage`, { accountName }));
+      uiLine();
+      process.exit(EXIT_CODES.ERROR);
+    }
+    return filteredPublicApps;
+  } catch (error) {
+    logger.debug(error);
```

Comment:
Should use `logErrorInstance` (or maybe `logApiErrorInstance`) here

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1052/commits/17da9c8463e4edcfc9166abc3ad864a30736e0dd